frontend/src/hooks/useSearchParamUpdater.ts

```diff
@@ -1,21 +1,20 @@
 "use client";
 
-import { usePathname } from "next/navigation";
+import { sendGAEvent } from "@next/third-parties/google";
+import { useSearchParams, usePathname, useRouter } from "next/navigation";
 
 export function useSearchParamUpdater() {
+  const searchParams = useSearchParams() || undefined;
   const pathname = usePathname() || "";
+  const router = useRouter();
+  const params = new URLSearchParams(searchParams);
 
-  // Singular string-type param updates include: search input, dropdown, and page numbers
-  // Multi/Set-type param updates include filters: Opportunity Status, Funding Instrument, Eligibility, Agency, Category
   const updateQueryParams = (
     queryParamValue: string | Set<string>,
     key: string,
+    queryTerm: string | null | undefined,
+    scroll = false,
```

Comment:
scroll is a query param?

Reply:
I documented the reason for this. It is implemented in the bottom pager so that when the user clicks the pager, they aren't left at the bottom of the screen when the results reload.